src/app/components/client/PetitionBanner.tsx

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { CloseBtn } from "../server/Icons";
+import { useL10n } from "../../hooks/l10n";
+import { useLocalDismissal } from "../../hooks/useLocalDismissal";
+import { useHasRenderedClientSide } from "../../hooks/useHasRenderedClientSide";
+import styles from "./PetitionBanner.module.scss";
+import { Button } from "./Button";
+import { CONST_URL_DATA_PRIVACY_PETITION_BANNER } from "../../../constants";
+
+export const PetitionBanner = () => {
+  const l10n = useL10n();
+
+  const hasRenderedClientSide = useHasRenderedClientSide();
+  const localDismissal = useLocalDismissal("data_privacy_petition_banner");
+
+  if (!hasRenderedClientSide || localDismissal.isDismissed) {
+    return null;
+  }
+
+  const { dismiss } = localDismissal;
+  return (
+    <div className={styles.banner}>
```

Comment:
For you consideration: might this make sense as an `<aside>`?

Reply:
I was wondering that as well. Especially, because we already have an `aside` for another banner. I feel like [aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside) might not be the correct tag for the content of this banner as it is not related to the content of the site:

> The \<aside\> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a portion of a document whose content is only indirectly related to the document's main content. Asides are frequently presented as sidebars or call-out boxes.